src/pudl/convert/epacems_to_parquet.py

```diff
@@ -88,20 +89,9 @@ def main():
     coloredlogs.install(fmt=log_format, level='INFO', logger=pudl_logger)
 
     args = parse_command_line(sys.argv)
-
-    # Make sure the requested years/states are available:
-    for year in args.years:
-        if year not in pc.WORKING_PARTITIONS["epacems"]["years"]:
-            raise ValueError(
-                f"{year} is not a valid year within the EPA CEMS dataset."
-            )
-    for state in args.states:
-        if state not in pc.WORKING_PARTITIONS["epacems"]["states"]:
-            raise ValueError(
-                f"{state} is not a valid state within the EPA CEMS dataset."
-            )
-
     pudl_settings = pudl.workspace.setup.get_defaults()
+    # This also validates the states / years we've been given:
+    epacems_settings = EpaCemsSettings(states=args.states, years=args.years)
```

Comment:
I switched this to using the validation that's part of the  `EpaCemsSettings` model instead of the duplicative code above.